package.json

```diff
@@ -66,7 +66,7 @@
     "node-fetch": "^2.6.1",
     "p-limit": "3.1.0",
     "react": "16.13.1",
-    "react-intl": "^2.8.0",
+    "react-intl": "^5.17.4",
```

Comment:
forgot to mention: we are now using exact versions for all our dependencies. Could you please set it to `"react-intl": "5.17.4"`?